@navikt/core/react/package.json

```diff
@@ -47,6 +48,7 @@
     "react-modal": "3.15.1"
   },
   "devDependencies": {
+    "@storybook/test-runner": "^0.11.0",
```

Comment:
should I only add the dependency to the innermost package.json in the monorepo?

Reply:
Yea, storybook-dependencies are defined @ root, so should probably be there to